### ISSUE TITLE
i am getting unexcepted start tag (body) how to resolve this

### ISSUE THREAD

**Issue by manjumenon** [validator/validator#630]
this is my html code

<!DOCTYPE html>
<html xmlns:th="http://www.thymeleaf.org">
<head>
  <title>Login</title>
</head>
<div th:replace="header :: meta"></div>
<div th:replace="header :: link"></div>
<body>
<div th:replace="header :: head"></div>
	<header class="masthead1">
		<div class="container">
			<div class="intro-text1 ">
				<div class="top-content">
					<div class="inner-bg">
						<div class="container">
							<div class="row"></div>
							<div class="row">
								<div class="col-sm-6 col-sm-offset-6 form-box">
									<div class="form-bottom">
										<div class="my-form mx-auto">
											<h3 class="">Login To our WebSite</h3>
											<p>Enter Your Email And password To Log On</p>
										</div>
										<div class="form-top-left"></div>
									</div>
									<div class="form-bottom">
										<form role="form" action="#" th:action="@{/login}"
											th:object="${loginDTO}" method="post" class="login-form"
											id="second_form">
											<div class="form-group">
												<label class="sr-only" for="form-username">UserID</label> <input
													type="text" name="userid" placeholder="UserId"
													class="form-username form-control" id="userid" />
											</div>

											
											<div class="form-group">
												<select class="form-control" name="email" id="email"
													required="required">
													<option value="">Select</option>
													<option value="1">@gmail.com</option>
													<option value="5">@yahoo.in</option>
													<option value="2">@outlook.com</option>
													<option value="3">@hotmail.com</option>
													<option value="4">@rediff.com</option>
												</select>
											</div>

											<div class="form-group">
												<label class="sr-only" for="form-password">Password</label>
												<input type="password" name="password"
													placeholder="Password" 
													class="form-password form-control" id="password" />
											</div>

											<button type="submit" class="btn">SignIN</button>

										</form>
									</div>
								</div>
							</div>
							<div class="row">
								<div class="col-sm-6 col-sm-offset-6 social-login">
									<h3>...or Register with:</h3>
									<div class="social-login-buttons">
										<a class="btn btn-link-1 btn-link-1-facebook" href="#"> <i
											class="fa fa-facebook"></i> Facebook
										</a> <a class="btn btn-link-1 btn-link-1-twitter" href="#"> <i
											class="fa fa-twitter"></i> Twitter
										</a> <a class="btn btn-link-1 btn-link-1-google-plus" href="#">
											<i class="fa fa-google-plus"></i> Google Plus
										</a>
									</div>
								</div>
							</div>
						</div>
					</div>

				</div>

			</div>
		</div>
	</header>

	
	<script src="./vendor/jquery/jquery.min.js"></script>
	<script src="./vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

	
	<script src="./vendor/jquery-easing/jquery.easing.min.js"></script>

	
	<script src="./js/jqBootstrapValidation.js"></script>
	<script src="./js/contact_me.js"></script>

	
	<script src="./js/agency.min.js"></script>

	<script
		src="https://cdn.jsdelivr.net/jquery.validation/1.16.0/jquery.validate.min.js"></script>

	<script type="text/javascript">
		$('form[id="second_form"]').validate({
			rules : {
				password : {
					required : true,
					minlength : 5,
				}
			},
			messages : {
				password : {
					minlength : 'Invalid userid and Password'
				}
			},
			submitHandler : function(form) {
				form.submit();
			}
		});

		/* $(function() {
				$( "#email" ).autocomplete({
				source: function( request, response ) {
				$.ajax({
				url: "http://localhost:8080/online/showemail/",
				dataType: "json",
				data: {
				maxRows: 10,
				startsWith: request.term
				},
				success: function( data ) {
				response( $.map( data.LoginEntity, function( item ) {
				return {
				label: item.userid + ", " +item.email + ", " + item.password,
				value: item.email
				}
				}));
				}
				});
				},
				minLength: 1,
				open: function() {
				$( this ).removeClass( "ui-corner-all" ).addClass( "ui-corner-top" );
				},
				close: function() {
				$( this ).removeClass( "ui-corner-top" ).addClass( "ui-corner-all" );
				}
				});
				});
		 */
	</script>
	
	<div th:replace="footer :: foot"></div>
	
	<div th:replace="footer :: scripts"></div>
</body>
</html>

**Comment by cvrebert** [comment]
```html
<!DOCTYPE html>
<html xmlns:th="http://www.thymeleaf.org">
<head>
  <title>Login</title>
</head>
<div th:replace="header :: meta"></div>
<div th:replace="header :: link"></div>
<body>
```

You have 2 `<div>`s that are outside of the `<body>`. They should instead be inside the `<body>`.

**Comment by sideshowbarker** [comment]
Closing as resolved per the explanation in https://github.com/validator/validator/issues/630#issuecomment-378085046
